ch03.adoc

```diff
@@ -295,7 +295,7 @@ The attributes **`flag_values`**,
 
 The **`flag_values`** and **`flag_meanings`**
       attributes describe a status flag consisting of mutually exclusive coded values. The **`flag_values`** attribute is the same type as the variable to which it is attached, and contains a list of the possible flag values. The **`flag_meanings`** attribute is a string whose value is a blank separated list of descriptive words or phrases, one for each flag value. Each word or phrase should consist of characters from the alphanumeric
-      set and the following five: '_', '-', '.', '+', '@'. If multi-word phrases are used to describe the flag values, then the words within a phrase should be connected with underscores. The following example illustrates
+      set and the following five: "_","-", ".","+", "@". If multi-word phrases are used to describe the flag values, then the words within a phrase should be connected with underscores. The following example illustrates
```

Comment:
I don't think this line needs changing.

1. Both the [HTML](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.6/build/cf-conventions.html#flags) and [PDF](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.6/build/cf-conventions.pdf) versions use a single quote character.
2. You've lost a couple of the spaces between items.